tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_rename_earmark_subvolume.py

```diff
@@ -50,7 +50,8 @@ def run(ceph_cluster, **kw):
         ran_string = "".join(random.choices("abcdefghijklmnopqrstuvwxyz", k=5))
         start_volume = f"cephfs1_{ran_string}"
         fs_util.create_fs(client1, start_volume)
-        log.info(client1.exec_command(sudo=True, cmd="ceph fs ls"))
+        out,rc = client1.exec_command(sudo=True, cmd="ceph fs ls -f json-pretty")
+        log.info(f"List of file systems : {out}")
```

Comment:
Please avoid using `format` strings in `log` statements. This is due to https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html